ci/licenses_golden/excluded_files

```diff
@@ -416,6 +416,7 @@
 ../../../flutter/sky/tools/create_embedder_framework.py
 ../../../flutter/sky/tools/create_full_ios_framework.py
 ../../../flutter/sky/tools/create_ios_framework.py
+../../../flutter/sky/tools/create_macos_binary.py
 ../../../flutter/sky/tools/create_macos_framework.py
 ../../../flutter/sky/tools/create_macos_gen_snapshots.py
```

Comment:
DBC

What's the difference between `create_macos_binary.py` and `create_macos_gen_snapshots.py`? Do we need both of them?

Reply:
Oh interesting -- create_macos_gen_snapshots is old and was used for iOS (and is hardcoded full of armv7 stuff); I guess we never cleaned it up?

I'll send out a patch to kill it.

Reply:
`create_macos_gen_snapshots` is used in the generator which processes the output of the builds

https://github.com/flutter/engine/blob/7a309e51270c23d4125cba0aa4fc6a7d19379433/ci/builders/mac_host_engine.json#L560

To avoid the timeout and because an hour is really quite long to wait for the builds to clear, maybe the snapshots should be created in different builds, and then processed with the generator at the end?

Reply:
Yes, there should be examples of using global generators already in the ios/macos build json files, and I think that can work here if we know how to split the different targets across different builders.